src/axom/sidre/core/Group.hpp

```diff
@@ -1179,14 +1179,39 @@ class Group
   /*!
    * \brief Load the Group from a file.
    *
-   * \param path      file path
-   * \param protocol  I/O protocol
+   * \param path     file path
+   * \param new_name the group name stored in the file
+   * \param protocol I/O protocol
    * \param preserve_contents   Preserve existing contents of group if true
    */
   void load(const std::string& path,
+            std::string & new_name,
```

Comment:
At our meeting there was some discussion of making the new_name argument optional, so that codes that don't care about the name can keep the old API.  I don't remember a consensus conclusion.

Reply:
I was going to ask the same thing. I think we agreed that ``load()`` should have two overloads, one with ``new_name`` and the other without, that way it is an optional parameter.

Reply:
You are right: my apologies: I'll make this change.

Reply:
As discussed, I made two overloads of each `load()` method, one with and one without the out-parameter.